src/components/PopupTrigger.js

```diff
@@ -72,17 +72,19 @@ const PopupTrigger = Utils.forwardRefWithName('PopupTrigger', ({ content, childr
 
 export default PopupTrigger
 
-export const InfoBox = ({ size, children, style, side }) => {
+export const InfoBox = ({ size, children, style, side, tooltip, iconOverride }) => {
   const [open, setOpen] = useState(false)
   return h(PopupTrigger, {
     side,
+    width: '100px',
```

Comment:
I'm _extremely_ hesitant to make a change like this to the base structure of something that's used in multiple places, rather than make it an optional addition.

Reply:
Oops, this wasn't meant to make it this far (and didn't work anyway), I'll remove it